### PR TITLE
Adding windows and mac migration command

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?

Adds the mac and windows agent configurations file migration command

### Motivation

it was missing

### CODE REVIEW

File: content/agent/_index.md [hunk 1]
```diff
@@ -64,7 +64,13 @@ To automatically transition between agent configuration paths and formats from A
 
 `sudo -u dd-agent -- datadog-agent import`
 
-The command parses an existing `datadog.conf` and convert all the bits that the new Agent still supports to the new `datadog.yaml` configuration file. It also copies configuration files for checks that are currently enabled.
+The command parses an existing `datadog.conf` and convert all the bits that the new Agent still supports to the new `datadog.yaml` configuration file. It also copies configuration files for checks that are currently enabled.
```
Comment (jeremy-lq) [inline content/agent/_index.md]: The command parses an existing `datadog.conf` and converts all bits the new Agent supports into the `datadog.yaml` configuration file. It also copies configuration files for checks that are currently enabled.

File: content/agent/_index.md [hunk 2]
```diff
@@ -64,7 +64,13 @@ To automatically transition between agent configuration paths and formats from A
...
 `sudo -u dd-agent -- datadog-agent import`
 
-The command parses an existing `datadog.conf` and convert all the bits that the new Agent still supports to the new `datadog.yaml` configuration file. It also copies configuration files for checks that are currently enabled.
+The command parses an existing `datadog.conf` and convert all the bits that the new Agent still supports to the new `datadog.yaml` configuration file. It also copies configuration files for checks that are currently enabled.  
+
+For Mac and Windows environment use:  
+
+`datadog-agent import <old_configuration_dir> <destination_dir>` 
+
+**Note**: On Windows, `datadog.conf` should be automatically upgraded to `datadog.yaml` on upgrade.
```
Comment (jeremy-lq) [inline content/agent/_index.md]: s/should be/is/

File: content/agent/_index.md (hunk 1 above)
Comment (l0k0ms) [reply content/agent/_index.md]: 👌

Comment (irabinovitch) [reply content/agent/_index.md]: is "all the bits" the right phrase here?

File: content/agent/_index.md (hunk 2 above)
Comment (l0k0ms) [reply content/agent/_index.md]: 👌

File: content/agent/_index.md [hunk 3]
```diff
@@ -64,7 +64,13 @@ To automatically transition between agent configuration paths and formats from A
 
 `sudo -u dd-agent -- datadog-agent import`
 
-The command parses an existing `datadog.conf` and convert all the bits that the new Agent still supports to the new `datadog.yaml` configuration file. It also copies configuration files for checks that are currently enabled.
+The command parses an existing `datadog.conf` and convert all the bits that the new Agent still supports to the new `datadog.yaml` configuration file. It also copies configuration files for checks that are currently enabled.  
+
+For Mac and Windows environment use:  
+
+`datadog-agent import <old_configuration_dir> <destination_dir>`
```
Comment (arbll) [inline content/agent/_index.md]: I think we should explain that `old_configuration_dir` is the directory containing the `datadog.conf` file and `destination_dir` the file where we will write the imported `datadog.yaml` (should be the same as `old_configuration_dir` on both windows & osx).

File: content/agent/_index.md [hunk 4]
```diff
@@ -64,7 +64,13 @@ To automatically transition between agent configuration paths and formats from A
 
 `sudo -u dd-agent -- datadog-agent import`
 
-The command parses an existing `datadog.conf` and convert all the bits that the new Agent still supports to the new `datadog.yaml` configuration file. It also copies configuration files for checks that are currently enabled.
+The command parses an existing `datadog.conf` and converts all bits the new Agent supports into the `datadog.yaml` configuration file. It also copies configuration files for checks that are currently enabled.  
+
+For Mac and Windows environment use:  
+
+`datadog-agent import <old_configuration_dir> <destination_dir>`
```
Comment (arbll) [inline content/agent/_index.md]: I think we should explain that `old_configuration_dir` is the directory containing the `datadog.conf` file and `destination_dir` the file where we will write the imported `datadog.yaml` (should be the same as `old_configuration_dir` on both windows & osx).

Comment (l0k0ms) [reply content/agent/_index.md]: 👌

File: content/agent/_index.md (hunk 1 above)
Comment (jeremy-lq) [reply content/agent/_index.md]: s/bits/configuration options/ does sound cleaner.

Comment (l0k0ms) [reply content/agent/_index.md]: 👌

**Pull request merged**